tensorflow/lite/swift/Sources/Model.swift

```diff
@@ -34,7 +40,8 @@ final class Model {
   }
 
   init?(modelData: Data) {
-    self.cModel = modelData.withUnsafeBytes { TfLiteModelCreate($0, modelData.count) }
+    self.data = modelData
+    self.cModel = self.data.withUnsafeBytes { TfLiteModelCreate($0, self.data.count) }
```

Comment:
Can you unwrap self.data since it is optional? Some tests are failing due to this. 

error: value of optional type 'Data?' must be unwrapped to refer to member 'withUnsafeBytes' of wrapped base type 'Data'
    self.cModel = self.data.withUnsafeBytes { TfLiteModelCreate($0, self.data.count) }

Reply:
Sorry somehow the merge is still blocked due to some check failures. Do you mind I patching this cl and submit it? Sorry it's been so long.

Reply:
No problem